docs/GettingStarted.md

```diff
@@ -119,23 +141,126 @@ Check the properties of the created stream. The **format** is one property which
+necessary, check stream properties before requesting start (especially if some
+stream parameters were left unspecified).
+```
+managedStream->requestStart();
+```
+In order to change the configuration of the stream, simply call this method
+again. The existing stream is closed, destroyed and a new stream is built and
+populates the managedStream.
+```
+// Modify the builder with some additional properties at runtime.
+builder.setDeviceId(MY_DEVICE_ID);
+// Re-open the stream with some additional config
+// The old ManagedStream is automatically closed and deleted
+builder.openManagedStream(managedStream);
+```
+The `ManagedStream` takes care of its own closure and destruction. If used in an
+automatic allocation context (such as a member of a class), the stream does not
+need to be closed or deleted. Make sure that the object which is responsible for
+the MangedStream (its enclosing class), goes out of scope when
+`Activity.onPause()` is called.
+
+The following class is a complete implementation of a ManagedStream, which
+renders a sine wave. Creating the class (e.g. through the JNI bridge) creates
+and opens an Oboe stream which renders audio, and its destruction stops and
+closes the stream.
+```
+#include <oboe/Oboe.h>
+#include <math.h>
+
+class OboeSinePlayer: public oboe::AudioStreamCallback {
+public:
+
+
+    OboeSinePlayer() {
+        oboe::AudioStreamBuilder builder;
+        // The builder set methods can be chained for convenience.
+        builder.setSharingMode(oboe::SharingMode::Exclusive)->setPerformanceMode(oboe::PerformanceMode::LowLatency);
+        builder.setChannelCount(kChannelCount)->setSampleRate(kSampleRate)->setFormat(oboe::AudioFormat::Float);
+        builder.setCallback(this)->openManagedStream(outStream);
+        // Typically, start the stream after querying some stream information, as well as some input from the user
+        outStream->requestStart();
+    }
+
+    oboe::DataCallbackResult onAudioReady(oboe::AudioStream *oboeStream, void *audioData, int32_t numFrames) override {
+        float *floatData = (float *) audioData;
+        for (int i = 0; i < numFrames; ++i) {
+            float sampleValue = kAmplitude * sinf(mPhase);
+            for (int j = 0; j < kChannelCount; j++) {
+                floatData[i * kChannelCount + j] = sampleValue;
+            }
+            mPhase += mPhaseIncrement;
+            if (mPhase >= kTWOPI) mPhase -= kTWOPI;
+        }
+        return oboe::DataCallbackResult::Continue;
+    }
+
+private:
+    oboe::ManagedStream outStream;
+    // Stream params
+    static int constexpr kChannelCount = 2;
+    static int constexpr kSampleRate = 48000;
+    // Wave params, these could be instance variables in order to modify at runtime
+    static float constexpr kAmplitude = 0.5f;
+    static float constexpr kFrequency = 440;
+    static float constexpr kPI = M_PI;
+    static float constexpr kTWOPI = kPI * 2;
```

Comment:
I was about to suggest using M_2_PI but then I looked it up and found it equals 2/pi !! Egad!

Consider kTwoPi